overlord/state/export_test.go

```diff
@@ -72,3 +72,15 @@ var (
 func (s *State) NumNotices() int {
 	return len(s.notices)
 }
+
+func MockGetTimeNow(f func() time.Time) (restore func()) {
+	old := timeNow
+	timeNow = f
+	return func() {
+		timeNow = old
+	}
+}
+
+func (n *Notice) GetLastOccurred() time.Time {
```

Comment:
No need to expose this for tests, I believe notices tests use `noticeToMap` to get unexported notice data.

Reply:
The problem is that `noticeToMap()` uses JSON marshalling to do its job, and one of the tests is to ensure that the dates are correct even after marshalling/unmarshalling...

Reply:
I agree with @ZeyadYasser. We're not testing JSON marshalling here, and JSON marshalling includes full nanoseconds (the resolution of `time.Duration`). So I think we should drop this and test via `noticeToMap` like everything else.

Reply:
Done.